scripts/wallet.js

```diff
@@ -404,7 +402,6 @@ export async function importWallet({
         }
 
         // For non-HD wallets: hide the 'new address' button, since these are essentially single-address MPW wallets
-        if (!wallet.isHD()) doms.domNewAddress.style.display = 'none';
 
         // Update the loaded address in the Dashboard
         wallet.getNewAddress({ updateGUI: true });
```

Comment:
my fault here on my refactor PR: it was meant to call the global function getNewAddress not the function getNewAddress of the wallet class